tools/rsdl/csdl-xml/lib/xmlSerializer.js

```diff
@@ -629,7 +628,7 @@ function writeAnnotation(xmlElement, term, annotationValue)
       break;
     break;
       default:
-        console.log("Annotation type " + typeof(annotationValue) + " not supported for term " + term);
+        console.log("Annotation type ${typeof(annotationValue)} not supported for term ${term}.");
```

Comment:
```suggestion
        console.log(`Annotation type ${typeof(annotationValue)} not supported for term ${term}.`);
```
Need to use backticks for string templates